### ISSUE TITLE
Classic Block: Consider to make the editor keyboard shortcuts visible in the tooltips

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#6195]
In WordPress, https://core.trac.wordpress.org/changeset/38897 made some editor keyboard shortcuts more discoverable by adding them to the buttons tooltips. See also https://core.trac.wordpress.org/ticket/43773

<img width="610" alt="screen shot 2018-04-16 at 10 19 59" src="https://user-images.githubusercontent.com/1682452/38800165-e1b0604e-4166-11e8-8cdc-313f01106833.png">

Instead, in Gutenberg the keyboard shortcuts are not exposed in the tooltips, not even the ones in the Classic editor (which is TinyMCE):

<img width="761" alt="screen shot 2018-04-16 at 10 33 09" src="https://user-images.githubusercontent.com/1682452/38800223-02e7fad8-4167-11e8-8deb-e17911a24311.png">

<img width="710" alt="screen shot 2018-04-16 at 10 33 19" src="https://user-images.githubusercontent.com/1682452/38800224-030664d2-4167-11e8-87e7-21ef62b2842c.png">

For more details on how the shortcuts are added and translated, see `src/wp-includes/js/tinymce/plugins/wordpress/plugin.js` in WordPress, which gets the translations from PHP via the localized editor settings object.

Aside:
notice the positioning (and size) of the tooltips in the classic editor block (TinyMCE) is different from the one in the Gutenberg blocks.

**Comment by afercia** [comment]
Some shortcuts are now displayed in the tooltips, see #6605 and #6669. The classic block doesn't show them yet, though.

**Comment by afercia** [comment]
I'd say this should be solved, as it's important for accessibility and also because there's no feature parity with the Classic Editor. /Cc @azaozz @iseulde

**Comment by Firestorm980** [comment]
I'd like to take this one (via @10up) - cc @tofumatt

**Comment by tofumatt** [comment]
That'd be awesome 👍

**Comment by mtias** [comment]
Is this in progress?

**Comment by tofumatt** [comment]
I don't see a PR for it; I've moved it out of the 5.0 RC milestone.

**Comment by Firestorm980** [comment]
Apologies @mtias @tofumatt, I haven't been able to make much headway on this one yet.

So far I have seen the behavior of a new classic block being added having the shortcuts, but on subsequent loads not having it. It also looked like shortcuts on TinyMCE were being removed since we appear to be handling them elsewhere. I'm wondering if this removal may be partly what's causing the issue and if so, how we'd work around that since TinyMCE is probably using that to show shortcuts in its tooltips.

**Comment by Firestorm980** [comment]
Fix in this PR: https://github.com/WordPress/gutenberg/pull/12447

**Comment by Firestorm980** [comment]
@tofumatt just want to make sure this is on your radar for the PR.

**Comment by tofumatt** [comment]
Hey @Firestorm980, really sorry this fell behind in my queue, but I've assigned myself to review that PR just now and should be looking shortly. Cheers for the ping and sorry it took me forever! 😓

**Comment by timwright12** [comment]
@tofumatt just checking in on this one

**Comment by afercia** [comment]
This has been fixed in core https://core.trac.wordpress.org/ticket/35710 by @azaozz. Thinking this issue and the pending PR #12447 can be closed now.

**Comment by aduth** [comment]
> This has been fixed in core https://core.trac.wordpress.org/ticket/35710 by @azaozz. Thinking this issue and the pending PR #12447 can be closed now.

In reviewing #12447 and tracing back to this note and related core commit and ticket, I'd conclude the same.
